docs/source/contribution_guidelines.rst

```diff
@@ -0,0 +1,127 @@
+Contribution Guidelines
+***********************
+
+Being open, we expect contributions from various walks of life. The below guidelines help in keeping the code base uniform irrespective of the person contributing to it.
```

Comment:
With this project being open source, it is important to have a uniform standard. The following guidelines will helps to have a uniform style, being it design or visual`.